File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/reserve/dtos/CreateReserveRequest.java
package com.slotify.backend.spring.reserve.dtos;

import jakarta.validation.constraints.NotNull;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
@ToString
public class CreateReserveRequest {

    @NotNull
    LocalDateTime dateTimeReserve;

    @NotNull
    UUID serviceId;

}

File: frontend/app/src/main/java/com/raj/slotify/dtos/reserves/CreateReserveRequest.kt
package com.raj.slotify.dtos.reserves

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime
import java.util.UUID

data class CreateReserveRequest(

    @SerializedName("dateTimeReserve")
    val dateTimeReserve: LocalDateTime,

    @SerializedName("serviceId")
    val serviceId: UUID

)
